src/EntityFramework/Core/EntityClient/EntityParameter.cs

```diff
@@ -319,10 +320,20 @@ public override object Value
                 }
 
                 _value = value;
+
+                _parameterStringValue = value.ToString() ?? null;
             }
         }
 
         // <summary>
+        // Gets the ToString() representation of the value of this parameter.
+        // </summary>
+        internal string ParameterStringValue
```

Comment:
Similar to a previous comment, this new field and parameter add public surface and overhead that doesn't seem to be used outside of the narrow scenario we are trying to fix. If we track a map of values to parameter instances while creating the schema queries (where we know that all parameter values are string) we shouldn't need this.